packages/11ty/_includes/components/lightbox/slides.js

```diff
@@ -43,7 +43,7 @@ module.exports = function(eleventyConfig) {
           case mediaType === 'soundcloud':
             return figureAudioElement(figure)
           case mediaType === 'table':
-            return await figureTableElement(figure)
+            return `<div class="table-wrapper">${await figureTableElement(figure)}</div>`
```

Comment:
@geealbers I think that this can be done without the wrapping element

```css
table {
  display: block;
  max-height: 80vh;
  max-width: 80vw;
  overflow: auto;
}
```

Though if there are reasons to use a wrapping element I would suggest a simple `tableContainter` class, for example

```css
.overflow-container > table {
  max-height: 80vh;
  max-width: 80vw;
  overflow: auto;
}
```

@anderspollack thoughts?

Reply:
I played around with it for a bit, and I think the wrapper is necessary--the `<table>` doesn't quite work with the rows like a generic container. but I agree that a more generalized `overflow-container` makes sense. Also currently, these table style selectors are one level more specific than they need to be. 

```
      table,
      caption {
        margin-bottom: 1rem;
      }
      table,
      th,
      td {
        border: 1px solid black;
        border-collapse: collapse;
        width: fit-content;
      }
      th,
      td {
        padding: 4px;
      }
      a {
        border-bottom: 1px dotted currentColor;
        color: currentColor;

        &:hover {
          border-bottom-style: solid;
        }
      }

```

are all nested under `.q-lightbox-slides__element--table .table-wrapper`, but I think it would be best if they were only nested under `.q-lightbox-slides__element--table` (styles specific to `.q-lightbox-slides__element--table .table-wrapper` are fine, but a more general `.overflow-container` utility class is better IMO)

Reply:
@anderspollack I'm cleaning this up per your comments and our discussion yesterday. I think `.overflow-container` would just have `overflow: scroll`, and the other properties here would be specific to the use of the overflow containe _for_ tables _in_ the lightbox slide? Is that right?

```scss
.overflow-container {
  overflow: scroll;
}

&--table .overflow-container {
  display: flex;
  flex-direction: column;
  max-height: 80vh;
  max-width: 80vw;
}
```

Reply:
Yep that sounds right to me! Although I might define the 
```
.overflow-container {
  overflow: scroll;
}
```
in `content/_assets/styles/utilities.scss` with the other utility classes like `visually-hidden`